Tests/Factory/DiactorosFactoryTest.php

```diff
@@ -111,7 +111,7 @@ public function testCreateRequest()
         $this->assertEquals(array('2.8'), $psrRequest->getHeader('X-Symfony'));
     }
 
-    public function testGetContentCanBeCalledAfterRequestCreation()
+    public function testGetContentCanBeCalledAfterRequestCreationWithStream()
```

Comment:
Maybe we can use a better speaking name like

testGetContentCanBeCalledAfterRequestCreationWithAManuallyPassedBody